warehouse/identities.go

```diff
@@ -420,15 +419,10 @@ func (wh *HandleT) populateHistoricIdentities(warehouse warehouseutils.Warehouse
 			panic(err)
 		}
 
-		job := UploadJobT{
-			upload:               &upload,
-			warehouse:            warehouse,
-			whManager:            whManager,
-			dbHandle:             wh.dbHandle,
-			pgNotifier:           &wh.notifier,
-			destinationValidator: validations.NewDestinationValidator(),
-			stats:                stats.Default,
-		}
+		job := wh.uploadJobFactory.NewUploadJob(&model.UploadJob{
+			Upload:    model.Upload(upload),
```

Comment:
Are we planning to remove [Upload](https://github.com/rudderlabs/rudder-server/blob/master/warehouse/upload.go#L69) struct in subsequent PR?